test/extended/apibinding/apibinding.go

```diff
@@ -0,0 +1,46 @@
+package apibinding
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area-apiexports]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-apibinding")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][API] Verify APIBinding working with personal workspace", func() {
+		// Shared compute could be only accessed from dev-provided test environments
+		// Skip for non-supported test environments
+		exutil.PreCheckEnvSupport(k, "kcp-stable.apps.kcp-internal", "kcp-unstable.apps.kcp-internal")
+		myWs := k.WorkSpace()
+
+		g.By("# Create an apibinding attach to the orgnaization workspace for the shared compute provided by acm should be failed")
```

Comment:
The syntax in short is like "Create xxx attach to xxx for xxx should xxx". The syntax in summary is like "verb1 xxx verb2 xxx should xxx". This sounds Chinese-English syntax :)
And typo of orgnaization.
How about:
```suggestion
		g.By("# Apibinding can't be used in an organization workspace")
```

Reply:
@Phaow @xingxingxia suggestion looks good to me here.

Reply:
Sure, thanks, I agree. I'll enhance it.